cafVecIjk.h: Add missing include

I am trying to build on Ubuntu 21.04 with gcc 10.3.0:
```
$ git clone git@github.com:OPM/ResInsight.git
$ cd ResInsight
$ mkdir build
$ cd build
$ cmake ..
-- The C compiler identification is GNU 10.3.0
-- The CXX compiler identification is GNU 10.3.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found OpenMP_C: -fopenmp (found version "4.5") 
-- Found OpenMP_CXX: -fopenmp (found version "4.5") 
-- Found OpenMP: TRUE (found version "4.5")  
-- Enabling OpenMP support
-- Could NOT find Octave (missing: OCTAVE_EXECUTABLE OCTAVE_ROOT_DIR OCTAVE_LIBRARIES) 
-- Found Git: /bin/git (found version "2.30.2") 
-- Found Linux
-- Setting build type to 'RelWithDebInfo' as none was specified.
-- Performing Test COMPILER_SUPPORTS_CXX11
-- Performing Test COMPILER_SUPPORTS_CXX11 - Success
-- Performing Test COMPILER_SUPPORTS_CXX0X
-- Performing Test COMPILER_SUPPORTS_CXX0X - Success
-- Found OpenMP_C: -fopenmp (found version "4.5") 
-- Found OpenMP_CXX: -fopenmp (found version "4.5") 
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE  
-- Found ZLIB: /usr/lib/x86_64-linux-gnu/libz.so (found version "1.2.11") 
-- Looking for access
-- Looking for access - found
-- Looking for _access
-- Looking for _access - not found
-- Looking for chdir
-- Looking for chdir - found
-- Looking for _chdir
-- Looking for _chdir - not found
-- Looking for chmod
-- Looking for chmod - found
-- Looking for fnmatch
-- Looking for fnmatch - found
-- Looking for fork
-- Looking for fork - found
-- Looking for fseeko
-- Looking for fseeko - found
-- Looking for fsync
-- Looking for fsync - found
-- Looking for ftruncate
-- Looking for ftruncate - found
-- Looking for getcwd
-- Looking for getcwd - found
-- Looking for _getcwd
-- Looking for _getcwd - not found
-- Looking for getpwuid
-- Looking for getpwuid - found
-- Looking for GetTempPath
-- Looking for GetTempPath - not found
-- Looking for getuid
-- Looking for getuid - found
-- Looking for glob
-- Looking for glob - found
-- Looking for gmtime_r
-- Looking for gmtime_r - found
-- Looking for localtime_r
-- Looking for localtime_r - found
-- Looking for lockf
-- Looking for lockf - found
-- Looking for mkdir
-- Looking for mkdir - found
-- Looking for _mkdir
-- Looking for _mkdir - not found
-- Looking for opendir
-- Looking for opendir - found
-- Looking for posix_spawn
-- Looking for posix_spawn - found
-- Looking for readlinkat
-- Looking for readlinkat - found
-- Looking for realpath
-- Looking for realpath - found
-- Looking for regexec
-- Looking for regexec - found
-- Looking for round
-- Looking for round - not found
-- Looking for symlink
-- Looking for symlink - found
-- Looking for timegm
-- Looking for timegm - found
-- Looking for usleep
-- Looking for usleep - found
-- Looking for unlink
-- Looking for unlink - found
-- Looking for _unlink
-- Looking for _unlink - not found
-- Looking for _tzname
-- Looking for _tzname - not found
-- Looking for tzname
-- Looking for tzname - found
-- Looking for execinfo.h
-- Looking for execinfo.h - found
-- Looking for getopt.h
-- Looking for getopt.h - found
-- Looking for unistd.h
-- Looking for unistd.h - found
-- Check if the system is big endian
-- Searching 16 bit integer
-- Looking for sys/types.h
-- Looking for sys/types.h - found
-- Looking for stdint.h
-- Looking for stdint.h - found
-- Looking for stddef.h
-- Looking for stddef.h - found
-- Check size of unsigned short
-- Check size of unsigned short - done
-- Searching 16 bit integer - Using unsigned short
-- Check if the system is big endian - little endian
-- Check size of time_t
-- Check size of time_t - done
-- Initializing GIT submodules
Submodule 'ThirdParty/qwt' (https://github.com/CeetronSolutions/qwt.git) registered for path 'ThirdParty/qwt'
Cloning into '/home/hakon/test/ResInsight/ResInsight/ThirdParty/qwt'...
Submodule path 'ThirdParty/qwt': checked out '0bd598c7008afbaa7216755fb9a4ac0da2e612eb'
-- No buildcache found
-- Found HDF5: /usr/lib/x86_64-linux-gnu/hdf5/serial/libhdf5_cpp.so;/usr/lib/x86_64-linux-gnu/hdf5/serial/libhdf5.so;/usr/lib/x86_64-linux-gnu/libpthread.so;/usr/lib/x86_64-linux-gnu/libsz.so;/usr/lib/x86_64-linux-gnu/libz.so;/usr/lib/x86_64-linux-gnu/libdl.so;/usr/lib/x86_64-linux-gnu/libm.so (found version "1.10.6") found components: CXX 
-- Using HDF5 libraries : /usr/lib/x86_64-linux-gnu/hdf5/serial/libhdf5_cpp.so;/usr/lib/x86_64-linux-gnu/hdf5/serial/libhdf5.so;/usr/lib/x86_64-linux-gnu/libpthread.so;/usr/lib/x86_64-linux-gnu/libsz.so;/usr/lib/x86_64-linux-gnu/libz.so;/usr/lib/x86_64-linux-gnu/libdl.so;/usr/lib/x86_64-linux-gnu/libm.so
-- Found Boost: /usr/lib/x86_64-linux-gnu/cmake/Boost-1.74.0/BoostConfig.cmake (found version "1.74.0") found components: filesystem system 
-- Found Boost: /usr/lib/x86_64-linux-gnu/cmake/Boost-1.74.0/BoostConfig.cmake (found version "1.74.0")  
-- Found Boost: /usr/lib/x86_64-linux-gnu/cmake/Boost-1.74.0/BoostConfig.cmake (found version "1.74.0") found components: filesystem system 
-- QtCharts found: Qt5::Charts
-- Found OpenGL: /usr/lib/x86_64-linux-gnu/libOpenGL.so   
-- Performing Test HAVE_GCC_SYNC_FUNCTIONS
-- Performing Test HAVE_GCC_SYNC_FUNCTIONS - Success
-- GCC synchronization functions detected
-- Could not find OCTAVE_MKOCTFILE, skipping OctavePlugin library
-- Configuring done
-- Generating done
-- Build files have been written to: /home/hakon/test/ResInsight/ResInsight/build

$ make
Scanning dependencies of target catch2
[  0%] Building CXX object ThirdParty/Ert/external/catch2/CMakeFiles/catch2.dir/dummy.cpp.o
[  0%] Linking CXX shared library ../../lib/libcatch2.so
[  0%] Built target catch2
Scanning dependencies of target ecl
[  1%] Building CXX object ThirdParty/Ert/lib/CMakeFiles/ecl.dir/util/rng.cpp.o
[  1%] Building CXX object ThirdParty/Ert/lib/CMakeFiles/ecl.dir/util/lookup_table.cpp.o
[  1%] Building CXX object ThirdParty/Ert/lib/CMakeFiles/ecl.dir/util/statistics.cpp.o
[  1%] Building CXX object ThirdParty/Ert/lib/CMakeFiles/ecl.dir/util/mzran.cpp.o
[  1%] Building CXX object ThirdParty/Ert/lib/CMakeFiles/ecl.dir/util/hash_node.cpp.o
[  1%] Building CXX object ThirdParty/Ert/lib/CMakeFiles/ecl.dir/util/hash_sll.cpp.o
[  1%] Building CXX object ThirdParty/Ert/lib/CMakeFiles/ecl.dir/util/path.cpp.o
[  1%] Building CXX object ThirdParty/Ert/lib/CMakeFiles/ecl.dir/util/hash.cpp.o
[  1%] Building CXX object ThirdParty/Ert/lib/CMakeFiles/ecl.dir/util/node_data.cpp.o
[  1%] Building CXX object ThirdParty/Ert/lib/CMakeFiles/ecl.dir/util/node_ctype.cpp.o
[  1%] Building C object ThirdParty/Ert/lib/CMakeFiles/ecl.dir/util/util.c.o
[  1%] Building CXX object ThirdParty/Ert/lib/CMakeFiles/ecl.dir/util/util_symlink.cpp.o
[  1%] Building C object ThirdParty/Ert/lib/CMakeFiles/ecl.dir/util/util_lfs.c.o
[  1%] Building CXX object ThirdParty/Ert/lib/CMakeFiles/ecl.dir/util/util_unlink.cpp.o
[  1%] Building CXX object ThirdParty/Ert/lib/CMakeFiles/ecl.dir/util/vector.cpp.o
[  1%] Building CXX object ThirdParty/Ert/lib/CMakeFiles/ecl.dir/util/parser.cpp.o
[  1%] Building CXX object ThirdParty/Ert/lib/CMakeFiles/ecl.dir/util/stringlist.cpp.o
[  1%] Building CXX object ThirdParty/Ert/lib/CMakeFiles/ecl.dir/util/buffer.cpp.o
[  1%] Building CXX object ThirdParty/Ert/lib/CMakeFiles/ecl.dir/util/timer.cpp.o
[  1%] Building CXX object ThirdParty/Ert/lib/CMakeFiles/ecl.dir/util/string_util.cpp.o
[  1%] Building CXX object ThirdParty/Ert/lib/CMakeFiles/ecl.dir/util/type_vector_functions.cpp.o
[  1%] Building CXX object ThirdParty/Ert/lib/CMakeFiles/ecl.dir/util/ecl_version.cpp.o
[  1%] Building CXX object ThirdParty/Ert/lib/CMakeFiles/ecl.dir/util/perm_vector.cpp.o
[  1%] Building CXX object ThirdParty/Ert/lib/CMakeFiles/ecl.dir/util/test_util.cpp.o
[  2%] Building CXX object ThirdParty/Ert/lib/CMakeFiles/ecl.dir/util/cxx_string_util.cpp.o
[  2%] Building C object ThirdParty/Ert/lib/CMakeFiles/ecl.dir/util/util_abort_gnu.c.o
[  2%] Building CXX object ThirdParty/Ert/lib/CMakeFiles/ecl.dir/util/test_work_area.cpp.o
[  2%] Building CXX object ThirdParty/Ert/lib/CMakeFiles/ecl.dir/util/util_getuid.cpp.o
[  2%] Building CXX object ThirdParty/Ert/lib/CMakeFiles/ecl.dir/util/util_opendir.cpp.o
[  2%] Building CXX object ThirdParty/Ert/lib/CMakeFiles/ecl.dir/util/util_spawn.cpp.o
[  2%] Building CXX object ThirdParty/Ert/lib/CMakeFiles/ecl.dir/util/util_lockf.cpp.o
[  2%] Building CXX object ThirdParty/Ert/lib/CMakeFiles/ecl.dir/util/path_stack.cpp.o

[...]

[ 38%] Building CXX object Fwk/AppFwk/CommonCode/CMakeFiles/CommonCode.dir/cafUtils.cpp.o
[ 38%] Building CXX object Fwk/AppFwk/CommonCode/CMakeFiles/CommonCode.dir/cvfStructGrid.cpp.o
[ 38%] Building CXX object Fwk/AppFwk/CommonCode/CMakeFiles/CommonCode.dir/cvfCellRange.cpp.o
[ 38%] Building CXX object Fwk/AppFwk/CommonCode/CMakeFiles/CommonCode.dir/cafColorTable.cpp.o
[ 38%] Building CXX object Fwk/AppFwk/CommonCode/CMakeFiles/CommonCode.dir/cafContourLines.cpp.o
[ 38%] Building CXX object Fwk/AppFwk/CommonCode/CMakeFiles/CommonCode.dir/cvfStructGridGeometryGenerator.cpp.o
[ 38%] Building CXX object Fwk/AppFwk/CommonCode/CMakeFiles/CommonCode.dir/cafVecIjk.cpp.o
In file included from /home/hakon/test/ResInsight/ResInsight/Fwk/AppFwk/CommonCode/cafVecIjk.cpp:37:
/home/hakon/test/ResInsight/ResInsight/Fwk/AppFwk/CommonCode/cafVecIjk.h:46:19: error: expected ‘)’ before ‘i’
   46 |     VecIjk( size_t i, size_t j, size_t k );
      |           ~       ^~
      |                   )
/home/hakon/test/ResInsight/ResInsight/Fwk/AppFwk/CommonCode/cafVecIjk.h:48:5: error: ‘size_t’ does not name a type
   48 |     size_t i() const;
      |     ^~~~~~
/home/hakon/test/ResInsight/ResInsight/Fwk/AppFwk/CommonCode/cafVecIjk.h:40:1: note: ‘size_t’ is defined in header ‘<cstddef>’; did you forget to ‘#include <cstddef>’?
   39 | #include <array>
  +++ |+#include <cstddef>
   40 | 
/home/hakon/test/ResInsight/ResInsight/Fwk/AppFwk/CommonCode/cafVecIjk.h:49:5: error: ‘size_t’ does not name a type
   49 |     size_t j() const;
      |     ^~~~~~
/home/hakon/test/ResInsight/ResInsight/Fwk/AppFwk/CommonCode/cafVecIjk.h:49:5: note: ‘size_t’ is defined in header ‘<cstddef>’; did you forget to ‘#include <cstddef>’?
/home/hakon/test/ResInsight/ResInsight/Fwk/AppFwk/CommonCode/cafVecIjk.h:50:5: error: ‘size_t’ does not name a type
   50 |     size_t k() const;
      |     ^~~~~~
/home/hakon/test/ResInsight/ResInsight/Fwk/AppFwk/CommonCode/cafVecIjk.h:50:5: note: ‘size_t’ is defined in header ‘<cstddef>’; did you forget to ‘#include <cstddef>’?
/home/hakon/test/ResInsight/ResInsight/Fwk/AppFwk/CommonCode/cafVecIjk.h:53:16: error: ‘size_t’ was not declared in this scope; did you mean ‘std::size_t’?
   53 |     std::array<size_t, 3> m_values;
      |                ^~~~~~
      |                std::size_t
In file included from /usr/include/c++/10/utility:68,
                 from /usr/include/c++/10/array:38,
                 from /home/hakon/test/ResInsight/ResInsight/Fwk/AppFwk/CommonCode/cafVecIjk.h:39,
                 from /home/hakon/test/ResInsight/ResInsight/Fwk/AppFwk/CommonCode/cafVecIjk.cpp:37:
/usr/include/x86_64-linux-gnu/c++/10/bits/c++config.h:264:26: note: ‘std::size_t’ declared here
  264 |   typedef __SIZE_TYPE__  size_t;
      |                          ^~~~~~
In file included from /home/hakon/test/ResInsight/ResInsight/Fwk/AppFwk/CommonCode/cafVecIjk.cpp:37:
/home/hakon/test/ResInsight/ResInsight/Fwk/AppFwk/CommonCode/cafVecIjk.h:53:25: error: template argument 1 is invalid
   53 |     std::array<size_t, 3> m_values;
      |                         ^
/home/hakon/test/ResInsight/ResInsight/Fwk/AppFwk/CommonCode/cafVecIjk.cpp:44:15: error: expected constructor, destructor, or type conversion before ‘(’ token
   44 | VecIjk::VecIjk( size_t i, size_t j, size_t k )
      |               ^
/home/hakon/test/ResInsight/ResInsight/Fwk/AppFwk/CommonCode/cafVecIjk.cpp:52:1: error: ‘size_t’ does not name a type
   52 | size_t VecIjk::i() const
      | ^~~~~~
/home/hakon/test/ResInsight/ResInsight/Fwk/AppFwk/CommonCode/cafVecIjk.cpp:38:1: note: ‘size_t’ is defined in header ‘<cstddef>’; did you forget to ‘#include <cstddef>’?
   37 | #include "cafVecIjk.h"
  +++ |+#include <cstddef>
   38 | 
/home/hakon/test/ResInsight/ResInsight/Fwk/AppFwk/CommonCode/cafVecIjk.cpp:60:1: error: ‘size_t’ does not name a type
   60 | size_t VecIjk::j() const
      | ^~~~~~
/home/hakon/test/ResInsight/ResInsight/Fwk/AppFwk/CommonCode/cafVecIjk.cpp:60:1: note: ‘size_t’ is defined in header ‘<cstddef>’; did you forget to ‘#include <cstddef>’?
/home/hakon/test/ResInsight/ResInsight/Fwk/AppFwk/CommonCode/cafVecIjk.cpp:68:1: error: ‘size_t’ does not name a type
   68 | size_t VecIjk::k() const
      | ^~~~~~
/home/hakon/test/ResInsight/ResInsight/Fwk/AppFwk/CommonCode/cafVecIjk.cpp:68:1: note: ‘size_t’ is defined in header ‘<cstddef>’; did you forget to ‘#include <cstddef>’?
make[2]: *** [Fwk/AppFwk/CommonCode/CMakeFiles/CommonCode.dir/build.make:229: Fwk/AppFwk/CommonCode/CMakeFiles/CommonCode.dir/cafVecIjk.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:1611: Fwk/AppFwk/CommonCode/CMakeFiles/CommonCode.dir/all] Error 2
make: *** [Makefile:171: all] Error 2
```

Any idea what is the problem?

--- a/Fwk/AppFwk/CommonCode/cafVecIjk.h
+++ b/Fwk/AppFwk/CommonCode/cafVecIjk.h
@@ -37,6 +37,7 @@
 #pragma once
 
 #include <array>
+#include <cstddef>
 
 namespace caf
 {

